save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -146,27 +186,52 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
                         }
                     }
                 }
-                doUploadChanges = {
-                    uploadChanges()
-                }
-                doReloadChanges = {
-                    reloadChanges()
-                }
-                doRunExecution = {
-                    runExecution()
-                }
+                doUploadChanges = ::uploadChanges
+                doReloadChanges = ::reloadChanges
+                doRunExecution = ::runExecution
+                doResultReload = ::resultReload
             }
         }
     }
 
     private fun ChildrenBuilder.renderDebugInfo() {
         state.debugInfo?.let { debugInfo ->
             div {
-                h6 {
-                    className = ClassName("text-center")
-                    +"DebugInfo"
+                val alertStyle = when (debugInfo.testStatus) {
+                    is Pass -> "success"
+                    is Fail -> "danger"
+                    is Ignored -> "secondary"
+                    is Crash -> "danger"
+                    else -> "info"
+                }
+                div {
+                    className = ClassName("alert alert-$alertStyle alert-dismissible fade show")
+                    role = AriaRole.alert
+                    div {
+                        displayTestResultDebugInfoStatus(debugInfo)
+                        a {
+                            href = "/#/sandbox"
```

Comment:
Could you try to use `Link { to }` here? We discussed that this is the preferred approach for react-router

Reply:
Removed `href` and marked the text In a different way - `strong` and `italic`